.github/workflows/shiny-react-test.yaml

```diff
@@ -16,10 +16,10 @@ jobs:
           node-version: 14
 
       - name: yarn install
-        run: yarn install
+        run: cd node && yarn install
```

Comment:
I used `working-directory: js` instead of `cd js` to increase readability. I'd rather not add a variable - additional complexity & less readability for me. LMK if you're OK with this, I will change it if you insist :slightly_smiling_face: